content/en/docs/reference/access-authn-authz/authentication.md

```diff
@@ -789,31 +789,43 @@ include metadata about the response.
 }
 ```
 
-The executed command is expected to print an `ExceCredential` to `stdout`. `k8s.io/client-go`
-will then use the returned bearer token in the `status` when authenticating against the
+The executed command is expected to print an `ExceCredential` to `stdout`.
+`k8s.io/client-go` will then use the returned bearer token and/or client TLS
+key and certificate from the `status` field when authenticating against the
 Kubernetes API.
 
+- The returned status may contain `token`, `clientCertificateData` and
+  `clientKeyData`, or all three
+- `clientCertificateData` and `clientKeyData` must be provided together
+- `clientCertificateData` may contain additional inermediate certificates to
+  send to the server
+
 ```json
 {
   "apiVersion": "client.authentication.k8s.io/v1alpha1",
   "kind": "ExecCredential",
   "status": {
     "token": "my-bearer-token"
```

Comment:
Missing a `,` at the end of this line and that's breaking the JSON

Reply:
Done